validators/docker_validator.go

```diff
@@ -63,16 +64,27 @@ func (d *DockerValidator) Validate(spec SysSpec) ([]error, []error) {
 
 	// Run 'docker info' with a JSON output and unmarshal it into a dockerInfo object
 	info := dockerInfo{}
-	out, err := exec.Command("docker", "info", "--format", "{{json .}}").CombinedOutput()
+	cmd := exec.Command("docker", "info", "--format", "{{json .}}")
+
+	// Stderr can contain warnings despite docker info success.
+	var outb, errb bytes.Buffer
+	cmd.Stdout = &outb
+	cmd.Stderr = &errb
+	err := cmd.Run()
 	if err != nil {
-		return nil, []error{errors.Errorf(`failed executing "docker info --format '{{json .}}'"\noutput: %s\nerror: %v`, string(out), err)}
+		return nil, []error{errors.Errorf(`failed executing "docker info --format '{{json .}}'"\noutput: %s\nstderr: %s\nerror: %v`, outb.String(), errb.String(), err)}
 	}
-	if err := d.unmarshalDockerInfo(out, &info); err != nil {
+	if err := d.unmarshalDockerInfo(outb.Bytes(), &info); err != nil {
 		return nil, []error{err}
 	}
 
 	// validate the resulted docker info object against the spec
-	return d.validateDockerInfo(spec.RuntimeSpec.DockerSpec, info)
+	warnings, errs := d.validateDockerInfo(spec.RuntimeSpec.DockerSpec, info)
+
+	if errb.String() != "" {
```

Comment:
```suggestion
	if len(errb.String()) > 0 {
```
this is just a styling change.